src/features/people/PeopleDetails/peopleDetailsSlice.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const peopleDetailsSlice = createSlice({
+    name: "personDetails",
```

Comment:
Hej! I found your mistake :)
to Redux the slice od state You are creating is called peopleDetails but then You put diferent value in name property.
So your Selector was asking for non existing slice 😄 

change that line to 
`name: "peopleDetails",`

you also have to correct it in line 30
💪

Reply:
See above for the detailed response. Thanks again! 😀🙏🌟